dces-drc-integration/src/integrationTest/java/uk/gov/justice/laa/crime/dces/integration/service/ContributionIntegrationTest.java

```diff
@@ -69,76 +68,136 @@ void testProcessContributionUpdateWhenFound() {
+     * <h4>Given:</h4>
+     * <p>* Update 3 concor_contribution records to the ACTIVE status for the purposes of the test.</p>
+     * <h4>When:</h4>
+     * <p>* The {@link ContributionService#processDailyFiles()} method is called.</p>
+     * <h4>Then:</h4>
+     * <p>1. The IDs of the 3 updated records are returned.</p>
+     * <p>2. The updated IDs are included in the list of IDs returned by `processDailyFiles`'s call to retrieve ACTIVE
+     *    concor_contributions.</p>
+     * <p>3. The updated IDs are included in the set of payloads sent to the DRC.</p>
+     * <p>4. A contribution file gets created, with at least 3 successful records and 0 failed records.<br>
+     *    The generated XML content includes each of the 3 linked rep_order IDs in a &lt;maat_id&gt; element tag.</p>
+     * <p>5. After the `processDailyFiles` method call returns, the concor_contribution entities corresponding to each
+     *    of the updated IDs is checked:<br>
+     *    - Each now has status SENT<br>
+     *    - Each now has a valid contribution_file ID<br>
+     *    - Each now has a last modified user of "DCES"<br>
+     *    - Each now has a last modified time during the method call.</p>
+     * <p>6. After the `processDailyFiles` method call returns, the contribution_file entity that was created is
+     *    checked:<br>
+     *    - It has the expected filename<br>
+     *    - It has a "records sent" of greater than or equal to 3<br>
+     *    - It has a created and last modified user of "DCES"<br>
+     *    - It has a created, sent and last modified time during the method call<br>
+     *    - The fetched XML content includes each of the 3 linked rep_order IDs in a &lt;maat_id&gt; element tag.</p>
+     *
+     * @see <a href="https://dsdmoj.atlassian.net/browse/DCES-349">DCES-349</a> for test specification.
+     */
     @Test
-    void testActiveContributionIsProcessed() {
+    void givenSomeActiveConcorContributions_whenProcessDailyFilesRuns_thenTheyAreQueriedSentAndInCreatedFile() {
+        // Update at least 3 concor_contribution rows to ACTIVE:
+        final var updatedIds = updateConcorContributionStatus(ConcorContributionStatus.ACTIVE, 3);
+
         final var watching = spyFactory.newContributionProcessSpyBuilder();
         watching.instrumentGetContributionsActive();
         watching.instrumentStubbedSendContributionUpdate(Boolean.TRUE);
         watching.instrumentUpdateContributions();
 
-        // Create at least 3 ACTIVE concor_contribution rows for this test to be meaningful:
-        final var updatedIds = updateConcorContributionStatus(ConcorContributionStatus.ACTIVE, 3);
-        watching.updatedIds(updatedIds);
-
         // Call the processDailyFiles() method under test (date range in case executing near to midnight)
         final var startDate = LocalDate.now();
         contributionService.processDailyFiles();
         final var endDate = LocalDate.now();
 
+        final var watched = watching.build();
```

Comment:
Since watched and watching.build() aren't every explicit, could we use an explicit type here for legibility.